debug/images.go

```diff
@@ -0,0 +1,87 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package debug
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+type DebugImagesManagement interface {
+	ShowDefaultImages() error
+	GetDefaultImage() (string, error)
+}
+const SealerRegistryUrl = "registry.cn-qingdao.aliyuncs.com/sealer-apps/"
```

Comment:
s/SealerRegistryUrl/DefaultSealerRegistryUrl/g

Reply:
Got it. With the help of golint, I modify this and other go code format.